src/screens/LoginPage/LoginPage.tsx

```diff
@@ -140,7 +140,7 @@ function LoginPage(): JSX.Element {
     ) {
       if (cPassword == signPassword) {
         try {
-          const { data } = await signup({
+          const { data: singUpData } = await signup({
```

Comment:
Updating the variable is an option, but the term `data` is too general in my opinion, and leaves ambiguity as to what data we are referring to. Providing an alias gives the same more context, and defines the variable in only the scope that it would be consumed.

Reply:
Yeah I agree, why not change the variable name instead?

Instead of
```...data: userData...```
Just use `userData`

Reply:
@Rumone It's cause we are using destructing here. By default, the `useQuery` hook returns the `data` object, which we can then alias to `userData`. If we were to access `userData` directly, the same would always be null as the same is never exported by the `useQuery` or analogous hooks.